site/en/docs/privacy-sandbox/fedcm/index.md

```diff
@@ -216,26 +242,598 @@ They can do the same for Chrome on desktop by going to
+
+### Create an IdP config file and endpoints {: #idp-config-file }
+
+The IdP config file provides a list of required endpoints for the browser. IdPs
+will host this config file and the required endpoints. All JSON response must be
+served with `application/json` content type.
+
+The config file's URL is determined by the values provided to the
+[`navigator.credentials.get` call executed on an RP](#sign-into-rp).
+
+```javascript
+const credential = await navigator.credentials.get({
+  identity: {
+    providers: [{
+      configURL: 'https://idp.example/anything.json',
+      clientId: '********',
+      nonce: '******'
+    }]
+  }
+});
+const { token } = credential;
+```
+
+Specify a full URL of the IdP config file location as a `configURL`. When
+[`navigator.credentials.get()` is called](#sign-into-rp) on the RP, the browser
+fetches the config file with a `GET` request without the `Referer` header. The
+request doesn't have cookies and doesn't follow redirects. This effectively
+prevents the IdP from learning who made the request and which RP is attempting
+to connect. For example:
+
+```http
+GET /anything.json HTTP/1.1
+Host: idp.example
+Accept: application/json
+Sec-Fetch-Dest: webidentity
+```
+
+{% Aside 'caution' %}
+
+All requests sent from the browser via FedCM include a `Sec-Fetch-Dest:
+webidentity` header to prevent [CSRF
+attacks](https://portswigger.net/web-security/csrf). All IdP endpoints must
+confirm this header is included.
+
+{% endAside %}
+
+The browser expects a JSON response from the IdP which includes the
+following properties:
+
+<table class="with-heading-tint with-borders">
+  <thead>
+    <tr>
+      <th>Property</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+  <tr>
+    <td><code>accounts_endpoint</code> (required)</td>
+    <td>URL for the <a href="#accounts-list-endpoint">accounts list endpoint</a>.</td>
+  </tr>
+  <tr>
+     <td><code>client_metadata_endpoint</code> (optional)</td>
+     <td>URL for the <a href="#client-metadata-endpoint">client metadata endpoint</a>.</td>
+  </tr>
+  <tr>
+     <td><code>id_assertion_endpoint</code> (required)</td>
+     <td>URL for the <a href="#id-assertion-endpoint">ID assertion endpoint</a>.</td>
+  </tr>
+  <tr>
+     <td><code>branding</code> (optional)</td>
+     <td>Object which contains various branding options.</td>
+  </tr>
+  <tr>
+     <td><code>branding.background_color</code> (optional)</td>
+     <td>Background color of the "Continue as..." button. Use the relevant CSS syntax, namely
+<a href="https://drafts.csswg.org/css-color-4/#typedef-hex-color"><code>hex-color</code></a>,
+<a href="https://drafts.csswg.org/css-color-5/#funcdef-hsl"><code>hsl()</code></a>,
+<a href="https://drafts.csswg.org/css-color-5/#funcdef-rgb"><code>rgb()</code></a>, or
+<a href="https://drafts.csswg.org/css-color-4/#typedef-named-color"><code>named-color</code></a>.</td>
+  </tr>
+  <tr>
+     <td><code>branding.color</code> (optional)</td>
```

Comment:
This should be the color of the text not the button. The button background is specified by branding.background_color